users/views.py

```diff
@@ -178,6 +178,11 @@ def get(self, request, *args, **kwargs):
                 # validated again in the parent get method.
                 pass
 
+        # Keycloak action passthrough
+        if request.GET.get('kc_action'):
```

Comment:
This could be walrusified := 
(double quotes would be nice in; `.get("kc_action")`

Reply:
Did some refactoring. :+1: